### PR TITLE
Allow excluding paths from scanning npm projects

### PR DESCRIPTION
**Pull request by yahavi**

### CODE REVIEW

File: src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form [hunk 1]
```diff
@@ -80,14 +75,57 @@
...
           <font name="Courier 10 Pitch"/>
           <text value=""/>
         </properties>
       </component>
+      <component id="1c8d6" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="4" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <text value="Npm exclusions"/>
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form]: Npm exclusions --> Npm excluded paths

File: src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form [hunk 2]
```diff
@@ -80,14 +75,57 @@
...
+          <toolTipText value="Glob pattern of excluded paths from Xray scan for npm."/>
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form]: Glob pattern of excluded paths from Xray scan for npm. -->
Pattern of project paths to exclude from Xray scanning for npm.

File: src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form [hunk 3]
```diff
@@ -80,14 +75,57 @@
...
+          <grid row="7" column="0" row-span="3" col-span="1" vsize-policy="6" hsize-policy="1" anchor="0" fill="2" indent="0" use-parent-layout="false"/>
+        </constraints>
+      </vspacer>
+      <component id="10519" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="5" column="1" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <enabled value="false"/>
+          <text value="A glob pattern used to exclude specific paths from being scanned by JFrog Xray."/>
```
Comment (eyalbe4) [inline src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form]: A glob pattern --> A global pattern

Comment (yahavi) [reply src/main/java/com/jfrog/ide/idea/ui/configuration/XrayGlobalConfiguration.form]: https://www.wikiwand.com/en/Glob_(programming)

**Pull request merged**